cockatrice/src/tab_game.cpp

```diff
@@ -309,6 +318,7 @@ TabGame::TabGame(TabSupervisor *_tabSupervisor, GameReplay *_replay)
     sayLabel(0),
     sayEdit(0)
 {
+    MainWindow = new QMainWindow;
```

Comment:
I don't know enough about the tools we're using to say.

@ctrlaltca is it possible to have valgrind target this line to check for leaks?